### PR TITLE
Code Refactoring and Reorganisation

### PR DESCRIPTION
**Pull request by ritik-aggarwal**

### CODE REVIEW

**Comment by syedsaadh** [comment]
1. Use Brime Notes specific content and Brime logo (https://github.com/BrimeNotes/android/raw/master/Logo-branding/logo_0.5x.png?raw=true) 
on the landing page
2. Footer should also contain respective content

File: package.json [hunk 1]
```diff
@@ -1,18 +1,19 @@
 {
-  "name": "webapp",
+  "name": "my-app",
```
Comment (imujjwal96) [inline package.json]: why do you keep changing this?

**Comment by syedsaadh** [comment]
@ritik-aggarwal Do resolve the above issue

**Comment by imujjwal96** [comment]
@/all can you show me a sample deployment.

**Pull request merged**